frontend/src/lib/components/ScriptBuilder.svelte

```diff
@@ -27,8 +27,9 @@
 	export let initialArgs: Record<string, any> = {}
 	export let lockedLanguage = false
 	export let topHash: string | undefined = undefined
+	export let showMeta: boolean = false
 
-	let metadataOpen = initialPath == '' && $page.url.searchParams.get('state') == undefined
+	let metadataOpen = showMeta || (initialPath == '' && $page.url.searchParams.get('state') == undefined)
```

Comment:
nit but I think snake casing is more common for query args although it doesn't seem to be a clear cut answer. My preference would be to use: `show_meta`

Reply:
Updated query param consumed on the page to be snake cased.